crates/starknet-devnet/Cargo.toml

```diff
@@ -48,3 +48,10 @@ starknet-rs-core = { workspace = true }
 starknet-rs-accounts = { workspace = true }
 hyper = { workspace = true }
 usc = { workspace = true }
+criterion = { version = "0.3.4", features = ["async_tokio"] }
+
+
+[[bench]]
+name = "mint_bench"
+harness = false
+path = "tests/benches/mint_bench.rs"
```

Comment:
Sorry I don't have experience with this kind of tests, can you leave more info on how you intended it to be executed?

Reply:
I'm executing it with cargo bench but as @FabijanC mentioned some documentation update would be helpful

Reply:
where should I place such documentation? as comments in the code, or in the README or create a seperate README within `benches/`?

Reply:
First put them in the PR description or this thread, then we'll see

Reply:
there is a Development section in readme.md it can be added as `Development - Memory Performance `